config/routes.rb

```diff
@@ -234,6 +234,16 @@
 
     get '/import-references' => 'import_references#index'
     post '/import-references' => 'import_references#import'
+
+    # https://github.com/mperham/sidekiq/wiki/Monitoring#rails-http-basic-auth-from-routes
+    require 'sidekiq/web'
+
+    Sidekiq::Web.use Rack::Auth::Basic do |username, password|
+      ActiveSupport::SecurityUtils.secure_compare(::Digest::SHA256.hexdigest(username), ::Digest::SHA256.hexdigest(ENV["SUPPORT_USERNAME"])) &
```

Comment:
Maybe use the abstracted versions of these creds provided by `BasicAuth`? (`initializers/basic_auth.rb`)